plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -412,4 +399,96 @@ public function get_name(): string {
 	public function get_description(): string {
 		return 'Synchronizes orders between posts and custom order tables.';
 	}
+
+	/**
+	 * Handle the 'deleted_post' action.
+	 *
+	 * When posts is authoritative and sync is enabled, deleting a post also deletes COT data.
+	 *
+	 * @param int     $postid The post id.
+	 * @param WP_Post $post The deleted post.
+	 */
+	private function handle_deleted_post( $postid, $post ): void {
```

Comment:
Would it make sense to specify params also in the function, not only in the comment? Same for other functions.
```suggestion
	private function handle_deleted_post( int $postid, \WP_Post $post ): void {
```

Reply:
The PR was merged with trunk so several changes was not related to the PR itself but to updated state of the trunk. Merged the feature branch to the trunk, so those changes won't be shown any more.

Reply:
nice, thanks!